jdbc/src/main/java/org/apache/iotdb/jdbc/IoTDBConnection.java

```diff
@@ -84,6 +84,13 @@
 
   private ZoneId zoneId;
   private boolean autoCommit;
+  private String url;
+
+  public String getUserName() {
+    return userName;
+  }
+
+  private String userName;
 
   public IoTDBConnection() {
```

Comment:
There is a method `getCatalog()` in this class. I think we can change the return value to the String `IOTDB`.